doc/topics/releases/2015.8.4.rst

```diff
@@ -0,0 +1,15 @@
+===========================
+Salt 2015.8.4 Release Notes
+===========================
+
+Core Changes
+============
+
+- Added ``state.validate`` feature.  This checks SLS data against the salt
+  state API and reports errors if the argument types do not match the types
```

Comment:
That's fine. I just want to make sure we thoroughly document the limitations of what we have here. I'm reluctant to merge a function named `state.validate` until we're comfortable that it can handle most, if not all, validation challenges.